### PR TITLE
Switch COS image to m85

### PR DESCRIPTION
**Pull request by vteratipally**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Switching to the latest COS LTS line.

**Which issue(s) this PR fixes**:Fixes #

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/config-default.sh [hunk 1]
```diff
@@ -83,7 +83,7 @@ fi
 # you are updating the os image versions, update this variable.
 # Also please update corresponding image for node e2e at:
```
Comment (karan) [inline cluster/gce/config-default.sh:84]: Also update https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml?

File: cluster/gce/config-default.sh [hunk 2]
```diff
@@ -83,7 +83,7 @@ fi
...
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
-GCI_VERSION=${KUBE_GCI_VERSION:-cos-81-12871-59-0}
+GCI_VERSION=${KUBE_GCI_VERSION:-cos-85-13310.1040.0}
```
Comment (bsdnet) [inline cluster/gce/config-default.sh:86]: it should cos-rc-85-13310-1040-0

File: cluster/gce/config-test.sh [hunk 3]
```diff
@@ -89,7 +89,7 @@ ALLOWED_NOTREADY_NODES=${ALLOWED_NOTREADY_NODES:-$(($(get-num-nodes) / 100))}
 # you are updating the os image versions, update this variable.
 # Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
-GCI_VERSION=${KUBE_GCI_VERSION:-cos-81-12871-59-0}
+GCI_VERSION=${KUBE_GCI_VERSION:-cos-85-13310.1040.0}
 export MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-}
```
Comment (bsdnet) [inline cluster/gce/config-test.sh:93]: Ditto cos-rc-85-13310-1040-0

File: cluster/gce/config-default.sh (hunk 1 above)
Comment (vteratipally) [reply cluster/gce/config-default.sh:84]: i am not able to access this file.

**Pull request closed without merging**